src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/MethodologyController.cs

```diff
@@ -9,15 +11,18 @@ namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Controllers
     [Produces(MediaTypeNames.Application.Json)]
     public class MethodologyController : ControllerBase
     {
-        public MethodologyController()
+        private readonly IMethodologyService _methodologyService;
+
+        public MethodologyController(IMethodologyService methodologyService)
         {
+            _methodologyService = methodologyService;
         }
 
         [HttpGet("methodologies/{slug}")]
         public async Task<ActionResult<MethodologyViewModel>> Get(string slug)
         {
-            // TODO SOW4 EES-2375 Return methodology from content database
-            return new NotFoundResult();
+            return await _methodologyService.Get(slug)
```

Comment:
Tempted to say rename `Get()` to something that reflects that it's getting the latest version of that Methodology that uses that slug, but not a biggie

Reply:
Thanks I've renamed this to GetLatestMethodologyBySlug now